games/Bash_Game.js

```diff
@@ -0,0 +1,750 @@
+........0.......
+........0.......
+........0.......
+........0.......
+........0.......
+........0.......
+........0.......
+................
+................`],
+  [n2, bitmap`
+................
+.....0000000....
+...........0....
+...........0....
+...........0....
+...........0....
+...........0....
+.....0000000....
+.....0..........
+.....0..........
+.....0..........
+.....0..........
+.....0..........
+.....0000000....
+................
+................`],
+  [n3, bitmap`
+................
+.....000000.....
+...........0....
+...........0....
+...........0....
+...........0....
+...........0....
+.....0000000....
+...........0....
+...........0....
+...........0....
+...........0....
+...........0....
+.....000000.....
+................
+................`],
+  [n4, bitmap`
+................
+................
+....0....0......
+....0....0......
+....0....0......
+....0....0......
+....0....0......
+....0....0......
+....000000000...
+.........0......
+.........0......
+.........0......
+.........0......
+.........0......
+................
+................`],
+    [cross, bitmap`
+................
+................
+................
+...00.......00..
+...000.....000..
+....000...000...
+.....000.000....
+......00000.....
+.......000......
+......00000.....
+.....000.000....
+....000...000...
+...000.....000..
+...00.......00..
+................
+................`],
+  [hl, bitmap`
+6..............6
+.6............6.
+..66.........6..
+...6........6...
+................
+................
+................
+6666............
+.............666
+................
+....66..........
+....6...........
+..66........6...
+..6..........6..
+.6............6.
+...............6`],
+  [confirm, bitmap`
+6.9.66....669..6
+.6.9666...699.69
+.969.66..699.69.
+..96..6..69.699.
+...66...6...69..
+....6......66...
+.66666....6.....
+66666...........
+.996.......66666
+99........666.9.
+....66.........9
+....69.........9
+..699.......6...
+..69........96..
+.699........9.6.
+..9.........99.6`],
+  [user, bitmap`
+0000000000000000
+0333333333333330
+0333300000033330
+0330000000000330
+0330000000000330
+0300222222220030
+0300002002000030
+0300002002000030
+0300002002000030
+0300000000000030
+0300220000220030
+0330022222200330
+0330000220000330
+0333300000033330
+0333333333333330
+0000000000000000`],
+  [oppo, bitmap`
+0000000000000000
+0555555555555550
+0555500000055550
+0550000000000550
+0550000000000550
+0500022000220050
+0500222202222050
+0502200202002050
+0500000000000050
+0500200000020050
+0500200000020050
+0550222002220550
+0550002222000550
+0555500000055550
+0555555555555550
+0000000000000000`],
+  [lose, bitmap`
+................
+................
+................
+...33.......33..
+...333.....333..
+....333...333...
+.....333.333....
+......33333.....
+.......333......
+......33333.....
+.....333.333....
+....333...333...
+...333.....333..
+...33.......33..
+................
+................`],
+)
+
+setSolids([arrow, coin, cross])
+
+let level = 9
```

Comment:
Right! So change this to 11:)

Reply:
Although I know I can change it to 11 after putting map 9 to the last the map sequences, is it fine for me to keep it at 9 because that's the default start level I want it to be at? I renamed it to startLevel now.

Or is this a convention thing?